migrations/86_notication_update.down.sql

```diff
@@ -0,0 +1,10 @@
+DROP INDEX unique_rule_name_per_project_rule_type;
```

Comment:
Maybe we should add IF EXISTS to avoid potential problems

Reply:
Fixed